### DISCUSSION TITLE
Practica de vacunación... otaku

### DISCUSSION THREAD

**Discussion by carlos-gmx3** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#403] [Show and tell]
Les comparto el resultado de la practica de CSS
quedo casi perfecta... solo que cambie algunas imágenes xD
ahora la misión es hacerla responsive
[MyVaccine](https://carlos-gmx3.github.io/Practicas3-CSS.FrontEnd/index.html)
Muchas partes fueron un reto del manejo de contenedores y algunos otros quedaron super bien con tablas.
Todo con tal de emular lo más posible la imagen de muestra

**Comment by RuthHernandez21** [comment]
@carlos-gmx3 Me encantó <3

**Comment by elrubiomorado** [comment]
Te quedo muy buena, yo igualmente estaba intentando hacerla otaku xd
